.github/workflows/release.yml

```diff
@@ -83,175 +83,260 @@ jobs:
-        git push origin
-        git push origin --tags
-    # Create GitHub Release
-    - name: Get release version changelog
-      id: get-changelog
-      if: ${{ steps.release-level.outputs.RELEASE_TAG == 'latest' }}
-      uses: superfaceai/release-changelog-action@v1
-      with:
-        path-to-changelog: CHANGELOG.md
-        version: ${{ steps.release-version.outputs.RELEASE_VERSION }}
-        operation: read
-    - name: Update GitHub release documentation
-      uses: softprops/action-gh-release@v1
-      if: ${{ steps.release-level.outputs.RELEASE_TAG == 'latest' }}
-      with:
-        tag_name: "js-v${{ steps.release-version.outputs.RELEASE_VERSION }}"
-        body: ${{ steps.get-changelog.outputs.changelog }}
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      # Setup
+      - uses: actions/checkout@v3
+      - name: Git configuration
+        run: |
+          git config --global user.email "bot@superface.ai"
+          git config --global user.name "GitHub Actions release workflow"
+      - uses: actions/setup-node@v3
+        with:
+          registry-url: https://registry.npmjs.org/
+          node-version: "18"
+          cache: yarn
+          cache-dependency-path: host/javascript/yarn.lock
+      # Release version
+      - name: Resolve release level
+        id: release-level
+        run: scripts/release-level.sh ${{ inputs.release-level }} ${{ inputs.release-kind }} >>$GITHUB_OUTPUT
+      - name: Resolve release version
+        id: release-version
+        run: scripts/release-version.sh ./host/javascript/VERSION ./host/javascript/VERSION ${{ steps.release-level.outputs.RELEASE_LEVEL }} ${{ steps.release-level.outputs.RELEASE_PREID }} >>$GITHUB_OUTPUT
+      # Build
+      - uses: actions/download-artifact@v3
+        with:
+          name: core-async-wasm
+          path: host/javascript/assets
+      - name: Copy LICENSE
+        run: cp LICENSE host/javascript/LICENSE
+      - name: Build host/javascript
+        working-directory: host/javascript
+        run: |
+          yarn install --frozen-lockfile
+          yarn build
+      # Changelog
+      - name: Update changelog
+        uses: superfaceai/release-changelog-action@v1
+        if: ${{ steps.release-level.outputs.RELEASE_TAG == 'latest' }}
+        with:
+          path-to-changelog: host/javascript/CHANGELOG.md
+          version: ${{ steps.release-version.outputs.RELEASE_VERSION }}
+          operation: release
+      # Commit release changes
+      - name: Git configuration
+        run: |
+          git config --global user.email "bot@superface.ai"
+          git config --global user.name "GitHub Actions release workflow"
+      - name: Commit package.json, VERSION, CHANGELOG.md and create git tag
+        working-directory: host/javascript
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git pull origin --rebase
```

Comment:
question: why rebase? I would imagine this would be `--ff-only`

Reply:
Ah, because I had it at the end before push and I wanted to keep linerity of commits.